.github/workflows/deploy-sdk.yml

```
@@ -42,6 +42,10 @@ jobs:
         uses: cycjimmy/semantic-release-action@v4
         id: semantic-release
         with:
+          # version numbers below can be in many forms: M, M.m, M.m.p
+          # version should be greater than the 22.0.1 (https://github.com/semantic-release/semantic-release/releases/tag/v22.0.1)
+          # because previous version had a bug in commit analyzer
+          semantic_version: 22.0.5
```

```suggestion
          semantic_version: latest
```

npm supports keyword `latest` to always get the latest version of the tool. 

I prefer we use `latest` instead of a hard-coded version so we can use future bug fixes introduced by the tool, too.